app/models/driver.rb

```diff
@@ -0,0 +1,49 @@
+require 'money'
+class Driver < ApplicationRecord
+  has_many :trips
+
+  validates :name, presence: true
+  validates :vin, presence: true
+
+  attribute :available, default: true
+
+  def mark_available
+    if self.available
+      self.available = false
+    else
+      self.available = true
+    end
+    self.save
+  end
```

Comment:
This is some interesting boolean logic-- I'd encourage y'all to think of ways to refactor this in the future! :)